corehq/apps/fixtures/templates/fixtures/view_table.html

```diff
@@ -20,6 +20,9 @@
 {% block page_content %}
   {% initial_page_data 'renderReportTables' 1 %}
   {% initial_page_data 'dataTablesOptions' data_tables_options %}
+  {% if not table_not_selected %}
+    <h4>{{ table_description }}</h4>
```

Comment:
It'd be nice to display the description on the manage tables page, too. Sticking it in a `.text-muted` div in the table cell for name would be easy.

Reply:
A slightly larger nice-to-have UI change would be for this to be editable from this page, using an inline edit widget, like apps/modules/forms in app manager. Adding the widget should be easy, but you'd need a dedicated ajax call to update the description, and I didn't look closely enough to see if that exists or if it would be annoying to add.

Reply:
I think I'll call that out of scope for this PR, so that this can be deployed before I go offline. If I have a chacne before Thursday, I'll PR adding the description there. (i'm expecting that handling text overflow will be annoying)